pkg/logs/processor/processor.go

```diff
@@ -18,27 +18,20 @@ import (
 // A Processor updates messages from an inputChan and pushes
 // in an outputChan
 type Processor struct {
-	inputChan    chan message.Message
-	outputChan   chan message.Message
-	apikey       string
-	logset       string
-	apikeyString []byte
```

Comment:
I would keep the `[]byte` version of `api_key/logset` to avoid computing it for each message

Reply:
done